src/main/java/edu/cornell/kfs/sys/CUKFSKeyConstants.java

```diff
@@ -165,8 +165,6 @@
     public static final String ERROR_CSACCOUNT_CONTINUATION_ACCOUNT_CLOSED = "error.csAccount.continuationAccount.closed";
     public static final String WARNING_CSACCOUNT_CONTINUATION_ACCOUNT_USED = "warning.csAccount.continuationAccount.used";
 
-    public static final String MESSAGE_DV_PAYEE_INVALID_PAYMENT_TYPE = "message.dv.payee.invalid.payment.type";
-
-    // KFSPTS-4792   
+    // KFSPTS-4792
```

Comment:
probably not, but there are a bunch of these unrelated to our contribution changes so I left them